draft-ietf-netmod-schedule-yang.md

```diff
@@ -172,19 +172,7 @@ System:
    * "recurrence-utc-with-periods" ({{sec-rec-utc-dt}})
    * "recurrence-time-zone-with-periods" ({{sec-rec-tz-dt}})
    * "icalendar-recurrence" ({{sec-ical-rec}})
-   * "schedule-status" and "schedule-status-with-name" ({{sec-schedule-status}})
-
-<!--
-   {{schedule-tree}} provides an overview of the tree structure of
-   the "ietf-schedule" module in terms of its groupings.
-
-~~~~
-{::include ./yang/tree/overall.txt}
-~~~~
-{: #schedule-tree title="Overall Schedule Tree Structure"}
-
-   Each of these groupings is presented in the following subsections.
--->
+   * "schedule-status" and "schedule-status-with-id" ({{sec-schedule-status}})
```

Comment:
I prefer we leave schedule-status-with-name but change schedule-id  to schedule-name to be consistent with "schedName" (MIB object)